src/main/scala/dpla/ingestion3/model/DplaMapData.scala

```diff
@@ -134,30 +104,18 @@ case class SkosConcept(
                       )
 
 case class DplaPlace(
-                     name: ZeroToOne[String] = None,
-                     city: ZeroToOne[String] = None,
-                     county: ZeroToOne[String] = None,
-                     state: ZeroToOne[String] = None,
-                     country: ZeroToOne[String] = None,
-                     region: ZeroToOne[String] = None,
-                     coordinates: ZeroToOne[String] = None
+                      name: ZeroToOne[String] = None,
+                      city: ZeroToOne[String] = None,
+                      county: ZeroToOne[String] = None,
+                      state: ZeroToOne[String] = None,
+                      country: ZeroToOne[String] = None,
+                      region: ZeroToOne[String] = None,
+                      coordinates: ZeroToOne[String] = None
                     )
 
 case class EdmTimeSpan(
-                        /* Why is this 0.n? I would have assumed ExactlyOne or ZeroToOne
-                            since it wouldn't make sense to have multiple original ranges
-                            per EdmTimeSpan but rather one edmTimeSpan per originalSourceDate?
-
-                            Esp. since in date in sourceResource is also ZeroToMany...
-
-                            I'm changing the type for the enrichment driver and need to follow-up
-                            w/team
-
-                            Also added a prefLabel property to store the enriched label
-                         */
                         originalSourceDate: ZeroToOne[String] = None,
-                        prefLabel: ZeroToOne[String] = None,
+                        prefLabel: ZeroToOne[String] = None, //todo not in spec?
```

Comment:
I thought we wanted a pref label for our enrichment generated date labels while preserving their original label?  It may not be in the spec but IIRC that was the change we discussed. Whether we should get ahead of the spec here is another question but the intent was clear AFAIK.

Reply:
I actually don't remember the conversation, actually. Just left a note to ask about it.

Reply:
So I thought about it, and I guess what I was wondering was, is our "enriched" model of a date EDTF, and represented in total by the begin and end fields, and a string-only representation redundant. My question is answered by the prefLabel field in MAP 4.1, so nevermind. :)